examples/multi-agent-debate/agents.py

```diff
@@ -0,0 +1,172 @@
+"""
+agents.py
+
+Defines agents for structured debates. Each function creates a ChatAgent 
+with specific configurations for its assigned role.
+"""
+
+import logging
+import langroid as lr
+from langroid.language_models import OpenAIGPTConfig
+
+# Set up a logger for this module
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def create_agent(base_llm_config: OpenAIGPTConfig, system_message: str) -> lr.ChatAgent:
+    """Creates a ChatAgent with a given system message and configuration.
+
+    Args:
+        base_llm_config (OpenAIGPTConfig): LLM configuration.
+        system_message (str): Role and guidelines for the agent.
+
+    Returns:
+        ChatAgent: Configured chat agent.
+    """
+    logger.info("Creating ChatAgent with system message: %s", system_message[:50])
+    config = lr.ChatAgentConfig(
+        llm=base_llm_config,
+        vecdb=None,
+        system_message=system_message,
+
+    )
+
+    return lr.ChatAgent(config)
+
+
+# Agents for AI in Healthcare
+def create_pro_ai_agent(base_llm_config: OpenAIGPTConfig) -> lr.ChatAgent:
```

Comment:
avoid defining a fn for each agent type. see later comment in main.py

Reply:
have a dict that holds the system messages, keyed by agent type

Reply:
done